src/NosCore.GameObject/Character.cs

```diff
@@ -1516,9 +1516,9 @@ public void RemoveVehicle()
             }
 
             IsVehicled = false;
-            VehicleSpeed = 0;
-            SendPacket(this.GenerateCond());
+            VehicleSpeed = null;
             MapInstance.Sessions.SendPacket(this.GenerateCMode());
+            SendPacket(this.GenerateCond());
```

Comment:
VehicleSpeed = null; Because if it isn't null it will keep sending the speed as 0.
Changed the order of packet generation to be like official